sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallMediaAsyncLiveTests.java

```diff
@@ -82,4 +85,75 @@ public void playMediaInACall(HttpClient httpClient) {
+    public void recognizeDtmfInACall(HttpClient httpClient) {
+        /* Test case: ACS to ACS call
+         * 1. create a CallAutomationClient.
+         * 2. create a call from source to one ACS target.
+         * 3. get updated call properties and check for the connected state.
+         * 4. prompt and recognize dtmf tones from target participant
+         * 4. hang up the call.
+         */
+
+        CallAutomationAsyncClient callClient = getCallAutomationClientUsingConnectionString(httpClient)
+            .addPolicy((context, next) -> logHeaders("recognizeDtmfInACall", next))
+            .buildAsyncClient();
+
+        CommunicationIdentityAsyncClient identityClient = getCommunicationIdentityClientUsingConnectionString(httpClient)
+            .addPolicy((context, next) -> logHeaders("recognizeDtmfInACall", next))
+            .buildAsyncClient();
+
+        try {
+            String callbackUrl = "https://localhost";
+            //String promptUrl = "https://localhost/audio/bot-hold-music-2.wav";
+            CommunicationIdentifier source = identityClient.createUser().block();
+            PhoneNumberIdentifier targetUser = new PhoneNumberIdentifier(PHONE_USER_1);
+            List<CommunicationIdentifier> targets = new ArrayList<>(Arrays.asList(targetUser));
+
+            CreateCallOptions createCallOptions = new CreateCallOptions(source, targets, callbackUrl)
+                .setSourceCallerId(ACS_RESOURCE_PHONE);
+
+            Response<CreateCallResult> callResponse = callClient.createCallWithResponse(createCallOptions).block();
+            assertNotNull(callResponse);
+            assertNotNull(callResponse.getValue());
+            assertNotNull(callResponse.getValue().getCallConnection());
+            assertNotNull(callResponse.getValue().getCallConnectionProperties());
+            waitForOperationCompletion(15000);
+
+            CallConnectionAsync callConnectionAsync = callClient.getCallConnectionAsync(callResponse.getValue().getCallConnectionProperties().getCallConnectionId());
+            assertNotNull(callConnectionAsync);
+            CallConnectionProperties callConnectionProperties = callConnectionAsync.getCallProperties().block();
+            assertNotNull(callConnectionProperties);
+            assertEquals(CallConnectionState.CONNECTED, callConnectionProperties.getCallConnectionState());
+
+            CallMediaAsync callMediaAsync = callConnectionAsync.getCallMediaAsync();
+
+            List<DtmfTone> stopTones = new ArrayList<>();
+            stopTones.add(DtmfTone.POUND);
+            CallMediaRecognizeDtmfOptions callMediaRecognizeDtmfOptions = new CallMediaRecognizeDtmfOptions(targetUser, 5)
+                .setStopTones(stopTones)
+                .setInterToneTimeout(Duration.ofSeconds(5));
+            callMediaRecognizeDtmfOptions.setInitialSilenceTimeout(Duration.ofSeconds(15));
+            //callMediaRecognizeDtmfOptions.setPlayPrompt(new FileSource().setUri(promptUrl));
```

Comment:
Is it intended to remove the play prompt part? If so, please remove this comment and also the comment out area above as well.